app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
- [ ] [OPTIONAL] It will be good to enable `:confirmable` for this project. After a user signs up, Devise sends a confirmation email containing a unique confirmation link. The user must click the confirmation link to verify their email address and activate their account. Only confirmed accounts are allowed to sign in. using "confirmable"  in Devise adds an important layer of security and improves the reliability of user registration by requiring email confirmation.